spec/lib/salus/scanners/language_version/ruby_version_scanner_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative '../../../../spec_helper.rb'
+
+describe Salus::Scanners::LanguageVersion::RubyVersionScanner do
+  describe '#run' do
+    context 'with non-ruby project' do
+      it 'should record error to scanner report' do
+        repo = Salus::Repo.new('spec/fixtures/blank_repository')
+        scanner = described_class.new(repository: repo, config: {})
+
+        expect(scanner.should_run?).to eq(false)
+
+        scanner.run
+
+        expect(scanner.report.passed?).to eq(false)
+
+        msg = scanner.report.to_h.fetch(:errors)[0][:message]
+        expect(msg).to include('Please supply the path to a ruby application')
+      end
+    end
+
+    context 'with ruby project' do
+      let(:path_str) { "../../../../../spec/fixtures/language_version/ruby_version_scanner" }
```

Comment:
Curios, will this resolve correctly: 'spec/fixtures/language_version/ruby_version_scanner' ?

Reply:
`path_str` should be resolved correctly. Because, the error messages([msg1](https://github.com/coinbase/salus/pull/493/files#diff-7047547931231d1e23293c450ae09215f8a2613e010171cb0f131839c454e2d5R54-R55), [msg2](https://github.com/coinbase/salus/pull/493/files#diff-7047547931231d1e23293c450ae09215f8a2613e010171cb0f131839c454e2d5R73-R74)) have been generated based on the version specified in the projects located in 'spec/fixtures/language_version/ruby_version_scanner'